pkg/gardenlet/controller/managedseed/managedseed_valueshelper_test.go

```diff
@@ -110,8 +110,7 @@ var _ = Describe("ValuesHelper", func() {
 				},
 			},
 			FeatureGates: map[string]bool{
-				string(features.Logging): true,
```

Comment:
Can you keep the existing test/scenario but just use a different feature gate? This tests seems to be related to merging of gardenlet configs, so obviously we don't want to drop a test case.

Reply:
Addressed in 6195a6338312c0bd41c0af2ca8a58ed1c7516dfc. PTAL

Reply:
Here we don't need it as there is already one feature gate present.

Reply:
I don't think so. So in the master branch of this test we have a feature gate (Logging) that is enabled in the parent gardenlet config and the same feature gate (Logging) disabled in the gardenlet config. And the test verifies that in the merged gardenlet config the feature gate value from the gardenlet config gets respected (parent config gets overwritten). With this change we drop exactly this case (there is another feature gate but it is with values true in both configs).